Source/JavaScriptCore/b3/B3LowerToAir.cpp

```diff
@@ -4469,6 +4469,25 @@ class LowerToAir {
         }
 
         case Select: {
+            if (m_value->type().isVector()) {
+                // Conditional moves aren't widely supported for vectors, so
+                // we lower vector Select to a branching construct.
+
+                auto ifTrueBlock = newBlock();
+                Air::BasicBlock* beginBlock;
+                Air::BasicBlock* doneBlock;
+                splitBlock(beginBlock, doneBlock);
+
+                append(Air::MoveVector, tmp(m_value->child(2)), tmp(m_value));
+                m_insts.last().append(createBranch(m_value->child(0)));
```

Comment:
nit: Can we use append() instead of  m_insts.last().append() for m_insts.last().append(createBranch(m_value->child(0)));? Maybe we need a refactoring patch to replace all m_insts.last().append() with append().

Reply:
Ah, I thought we couldn't do so, but it looks like we can. I'll change it!

Reply:
Changed in latest push.